python/GafferSceneUI/LightEditor.py

```diff
@@ -192,9 +224,19 @@ def _titleFormat( self ) :
 	@GafferUI.LazyMethod()
 	def __updateColumns( self ) :
 
-		sections = self.__columnRegistry[self.__settingsNode["attribute"].getValue()]
-		section = sections[self.__settingsNode["section"].getValue() or None]
-		sectionColumns = [ c( self.__settingsNode["in"], self.__settingsNode["editScope"] ) for c in section.values() ]
+		# sections = self.__columnRegistry[self.__settingsNode["attribute"].getValue()]
+		# section = sections[self.__settingsNode["section"].getValue() or None]
+		# sectionColumns = [ c( self.__settingsNode["in"], self.__settingsNode["editScope"] ) for c in section.values() ]
+
```

Comment:
Remove dead code.

Reply:
Removed in 809667e83655070ce1554bc6dc5b93fdfb8f40ef